test/summit_run_tests.lsf

```diff
@@ -36,15 +34,15 @@ source build-aux/summit_load_modules.sh
 BUILD_DIR="build_$LSB_JOBID"
 mkdir $BUILD_DIR && cd $BUILD_DIR
 
-# Run CMake to configure the build (Default: Debug).
-type=${DCA_BUILD_TYPE:-Debug}
-echo "DCA build type: ${type}"
-cmake -C ../build-aux/summit.cmake -DCMAKE_BUILD_TYPE=$type -DDCA_WITH_TESTS_FAST=ON -DDCA_WITH_TESTS_EXTENSIVE=ON ../
+# Run CMake to configure the build.
+build_type=${DCA_BUILD_TYPE:-Debug}
+printf "Build type: ${build_type}\n\n"
+cmake -C ../build-aux/summit.cmake -DCMAKE_BUILD_TYPE=$build_type -DDCA_WITH_TESTS_FAST=ON -DDCA_WITH_TESTS_EXTENSIVE=ON ../
 
 # Build the tests.
 make -j
```

Comment:
I think output on failure is a much more useful option to understand what happened without having to manually rerun the tests.

Reply:
The detailed output is always written to the test log, which is located in build/Testing/Temporary and for Jenkins runs copied to our project dir.
I don't think it's useful to clutter the test summary with test outputs. It will be quite a mess if you have a test failing that checks a function with thousands of elements.

Reply:
I see, that is quite useful to know, thanks! But the log file still contains all the test output, and I believe it is quite easier to obtain the important information with output-on-failure, especially as it still produces a compact summary at the end of the log, easily accessible with the tail command. Also having the relevant info in the Jenkins email would save the time to log into the cluster find the correct log, unless there is a problem with having too much text dumped it the email.